src/app/InteractionModelEngine.cpp

```diff
@@ -1853,26 +1853,21 @@ void InteractionModelEngine::ResumeSubscriptionsTimerCallback(System::Layer * ap
             continue;
         }
 
-        auto requestedAttributePathCount = subscriptionInfo.mAttributePaths.AllocatedSize();
-        auto requestedEventPathCount     = subscriptionInfo.mEventPaths.AllocatedSize();
-        if (!imEngine->EnsureResourceForSubscription(subscriptionInfo.mFabricIndex, requestedAttributePathCount,
-                                                     requestedEventPathCount))
+        auto subscriptionResumptionHelper = Platform::MakeUnique<SubscriptionResumptionHelper>();
```

Comment:
Should you be using `Platform::New` and `Platform::Delete` now since you are doing the management yourself?

Reply:
Done